.github/workflows/windows.yml

```diff
@@ -95,15 +95,15 @@ jobs:
 # Enable it, when working on the installer.
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
       - name: Checkout IrrlichtMt
         run: |
           $ref = @(Get-Content misc\irrlichtmt_tag.txt)
           git clone https://github.com/minetest/irrlicht lib\irrlichtmt --depth 1 -b $ref[0]
 
       - name: Restore from cache and run vcpkg
-        uses: lukka/run-vcpkg@v7
+        uses: lukka/run-vcpkg@v11.1
```

Comment:
upgrading this one should be possible but it looks like you'll have to move the arguments into a `vcpkg.json`
(please don't put it into the root folder tho)

Reply:
updated the pr up top to denote out of scope